src/file/FileController.ts

```diff
@@ -320,6 +320,11 @@ export class FileController {
 		}
 	}
 
+	async saveStringAsFile(contents: string, name: string, mimeType: string) {
```

Comment:
I don't know if it's justified to have this function, we operate on files elsewhere in this class. I'm not insisting tho.

Reply:
i like convenience methods